spread/build/fedora/task.yaml

```diff
@@ -0,0 +1,47 @@
+summary: Build (on Fedora)
+systems: [fedora-27]
+variants: 
+
+
+execute: |
+    dnf install --assumeyes \
+        cmake \
+        gcc-c++ \
+        boost-devel \
+        mesa-libEGL-devel \
+        mesa-libGLES-devel \
+        glm-devel \
+        protobuf-lite-devel \
+        protobuf-compiler \
+        capnproto-devel \
+        capnproto \
+        glog-devel \
+        gflags-devel \
+        systemd-devel \
+        glib2-devel \
+        wayland-devel \
+        mesa-libgbm-devel \
+        libepoxy-devel \
+        nettle-devel \
+        libinput-devel \
+        libxml++-devel \
+        libuuid-devel \
+        libxkbcommon-devel \
+        freetype-devel \
+        lttng-ust-devel \
+        libatomic \
+        qterminal \
+        qt5-qtwayland \
+        python3-pillow \
+        libevdev-devel \
+        umockdev-devel \
+        gtest-devel \
+        gmock-devel \
+        ninja-build
+
+    cd $SPREAD_PATH
+    cd $(mktemp --directory)
+    cmake $SPREAD_PATH -DCMAKE_BUILD_TYPE=Debug -DMIR_USE_LD_GOLD=ON -G Ninja
+    ninja
+    ninja ptest
```

Comment:
Having discovered `cmake -H<source_dir> -B<build_dir>`, think we could use that along with `cmake --build<build_dir>`?